appeals/api/src/server/endpoints/representations/representations.service.js

```diff
@@ -258,5 +264,85 @@ export async function publishFinalComments(appeal, azureAdUserId) {
 		VALIDATION_OUTCOME_COMPLETE
 	);
 
+	try {
+		if (
+			result.some((rep) => rep.representationType === APPEAL_REPRESENTATION_TYPE.LPA_FINAL_COMMENT)
+		) {
+			notifyLpaFinalComments(appeal, notifyClient);
+		}
+		if (
+			result.some(
+				(rep) => rep.representationType === APPEAL_REPRESENTATION_TYPE.APPELLANT_FINAL_COMMENT
+			)
+		) {
+			notifyAppellantFinalComments(appeal, notifyClient);
+		}
+	} catch (error) {
+		logger.error(error);
+	}
+
 	return result;
 }
+
+/**
+ * @param {Appeal} appeal
+ * @param {import('#endpoints/appeals.js').NotifyClient} notifyClient
+ * @param {import('#endpoints/appeals.js').NotifyTemplate} template
+ * @param {string} recipientEmail
+ * */
+async function notifyFinalComments(appeal, notifyClient, template, recipientEmail) {
```

Comment:
Could these functions maybe be called `notifyFinalCommentsPublished` or something similar? Just so it's clear when we expect this to fire